tests/unit/s2n_async_pkcs11_sign_test.c

```diff
@@ -0,0 +1,574 @@
+/*
```

Comment:
FYI I also have an implementation that uses OpenSSL's Engine system to do the sigantures. It came out a little cleaner / smaller then this test, which directly uses PKCS \#11, happy to swap the two or commit both if there is any preference.

Reply:
Awesome, will work towards having both integrated